src/moonlight-css/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://moonlight-mod.github.io/manifest.schema.json",
   "id": "moonlight-css",
-  "version": "2.0.3",
+  "version": "2.1.0",
```

Comment:
Doesn't feel significant enough to bump to 2.1.0 but thats just me (considering what I've added before would've ended up being 2.1.0 then and would make this 2.2.0)

Versioning is annoying